src/sider/senoppfolging/SenOppfolgingSide.tsx

```diff
@@ -2,16 +2,17 @@ import React, { ReactElement } from "react";
 import Side from "@/sider/Side";
 import { Menypunkter } from "@/components/globalnavigasjon/GlobalNavigasjon";
 import Sidetopp from "@/components/Sidetopp";
+import SenOppfolging from "@/sider/senoppfolging/SenOppfolging";
 
 const texts = {
   title: "Oppfølging i sen fase",
 };
 
-export const SenOppfolgingSide = (): ReactElement => {
+export default function SenOppfolgingSide(): ReactElement {
   return (
     <Side tittel={texts.title} aktivtMenypunkt={Menypunkter.SENOPPFOLGING}>
       <Sidetopp tittel={texts.title} />
-      <div>Her kommer det ting!</div>
+      <SenOppfolging />
     </Side>
   );
-};
+}
```

Comment:
Er det litt rart at dette skal være egen side? 🤔

Reply:
Ja per nå er det kanskje det. Jeg la inn tredeling og sånt, men fjernet det for PRen, og da lå denne igjen alene så det ser kanskje litt mer wonky ut nå. Men tenker at innhenting av svarene og sånt fint kan ligge i en egen komponent og ikke blandes sammen med den funksjonaliteten som omhandler sideoppsettet.